client/src/components/section2/Location.jsx

```diff
@@ -0,0 +1,47 @@
+import React, {Component} from 'react';
+import styled from 'styled-components';
+import { MapPin } from 'react-feather';
+
+class Location extends Component {
+    constructor(props) {
+        super(props)
+    }
+
+    render() {
+        let randomIdx = Math.floor(Math.random() * 100)
```

Comment:
I think this name could cause errors if you're rewriting it. Might just be my preference. @dekim2324